### PR TITLE
[Merged by Bors] - Modify benchmark README.md

### LINKED ISSUE #209: Automatic test by benchmark error
error log：
root@43e85027b36c:/home/john/Adlik# cat serving_time.log
2020-07-08 06:04:11: I adlik_serving/framework/manager/runtime_suite.cc:18] found runtime tensorflow
2020-07-08 06:04:11: I adlik_serving/server/main.cc:11] Adlik serving version: 1.0.0-dev
2020-07-08 06:04:11: I adlik_serving/server/core/server_core.cc:50] Adlik serving is initializing...
2020-07-08 06:04:11: I adlik_serving/runtime/tensorflow/factory/tf_runtime.cc:237] initialize tensorflow runtime
2020-07-08 06:04:11: I adlik_serving/runtime/tensorflow/factory/tf_runtime.cc:242] configure tensorflow runtime
2020-07-08 06:04:11: I adlik_serving/framework/manager/model_factory_suite.cc:14] register model factory:tensorflow
2020-07-08 06:04:11: I adlik_serving/framework/domain/model_store.cc:20] model root path: /home/john/Adlik/model/model_repos
2020-07-08 06:04:11: E cub/env/posix/posix_filesystem.cc:154] assertion failed: unexpected null ptr: dir
2020-07-08 06:04:11: E adlik_serving/server/core/server_core.cc:90] static_cast<ModelStore&>(*this).config() got 4294967295
2020-07-08 06:04:11: E adlik_serving/server/core/server_core.cc:52] config() got 4294967295
root@43e85027b36c:/home/john/Adlik# cd /home/john/Adlik/model/model_repos
bash: cd: /home/john/Adlik/model/model_repos: No such file or directory

not figure out the  relationship between "export_path" in serving_model.json and "--model_base_path" in serving_script.sh

### PR DESCRIPTION
**Pull request by XueweiHan21**
Closes #209.

Signed-off-by: hanxuewei <han.xuewei@zte.com.cn>

### CODE REVIEW

File: benchmark/README.md [hunk 1]
```diff
@@ -29,19 +29,19 @@ The parameters of the automatic test framework are as follows:
 If you want to use the automatic_test.py test the runtime, you need to follow the steps below:
 
 1. Download Adlik code.
-2. Install docker.
-3. Prepare the serving_model.json (required for compile model) trained model, the format of the model file can be: .pb,
+2. Prepare the serving_model.json (required for compile model) trained model, the format of the model file can be: .pb,
```
Comment (amadeus-zte) [inline benchmark/README.md:32]: typo :  for compiling model?

**Comment by amadeus-zte** [comment]
bors retry

**Comment by EFanZh** [comment]
bors retry

**Pull request closed without merging**